Dockerfile

```diff
@@ -1,17 +1,16 @@
 # syntax=docker/dockerfile:1
 
-ARG GO_VERSION=1.18
-
-ARG ALPINE_VERSION=3.16
-ARG ALPINE_BASE=alpine:${ALPINE_VERSION}
+ARG GO_VERSION=1.19
```

Comment:
Not 1.20?

Reply:
Ah yes it was not GA when I made this changes last year. Updated to 1.20.